src/partials/footer.html

```diff
@@ -1 +1,30 @@
-<footer>TEST</footer>
+<footer>
+  <div>
+    <div>HOMEMADE ICE CREAM MADE<span> WITH PASSION</span></div>
+    <div>
+      <ul>
+        <li>
+          <a target="_blank"
+            ><svg><use>git</use></svg></a
+          >
+        </li>
+        <li>
+          <a target="_blank"
+            ><svg><use>tel</use></svg></a
+          >
+        </li>
+        <li>
+          <a target="_blank"
+            ><svg><use>fb</use></svg></a
+          >
+        </li>
+        <li>
+          <a target="_blank" href="tel.+610383766284"
```

Comment:
Tutaj nie używasz target.
Poprawny format tel to taki: href="tel:+48111111111"

Reply:
no okej, dla mnie target="_blank mówi tylko o tym, że ma się otworzyć w nowej karcie, więc w sumie nie za wiele zmienia, href przecież jest, ale ok, wywale.

Reply:
czemu taki href, a nie z numerem telefonu z makiety mam dać?

Reply:
dałem przykład, chodziło mi o to że zamiast dwukropka po tel dałaś kropkę